cypress/integration/singleCryptid-test.js

```diff
@@ -0,0 +1,27 @@
+describe('All sightings page cypress test', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+  
+  it('Should confirm that true is equal to true', () => {
+    expect(true).to.equal(true)
+    expect(true).to.equal(true)
+  });
+
+  it('Should start on the home page and click on a cryptid', () => {
+    cy.get('\#\\32').click()
+  })
+
+  it('Should view the cryptids information page', ()=> {
```

Comment:
Line 13 and 17 are really cool. Also like how detailed this is to check that it's navigating to a new page and making sure that everything is rendering.  Nice work